python/lsst/faro/measurement/TractTableMeasurementTasks.py

```diff
@@ -91,18 +150,22 @@ class TExTableTask(Task):
     _DefaultName = "TExTableTask"
 
     def run(
-        self, metricName, catalog, band=None
+        self, metricName, catalog, currentBand, **kwargs
     ):
 
         self.log.info("Measuring %s", metricName)
 
         # If accessing objectTable_tract, we need to append the band name at
         #   the beginning of the column name.
-        if band is not None:
-            prependString = band
+        if currentBand is not None:
+            prependString = currentBand
         else:
             prependString = None
 
+        # filter catalog
+        catalog = selectors.applySelectors(catalog,
```

Comment:
So does this fully address the to-do in lines 585-591 of tex_table.py (https://github.com/lsst/faro/blob/tickets/DM-32611/python/lsst/faro/utils/tex_table.py#L585-L591)? I think it does, just want to be sure. I think it's probably worth deleting those lines as part of this PR if that's the case.

Reply:
It does and I have deleted those lines.